includes/wikia/VariablesBase.php

```diff
@@ -2637,6 +2637,68 @@
  */
 $wgFandomAppSmartBannerText = null;
 
+/**
+ * Used to display fandom store names by wikiId for IntentX
```

Comment:
let's add infor that is this prod :)

Reply:
I think you're talking about comments here and not variable names?

Reply:
Yes, comment. Wiki IDs are different on those two environments

Reply:
Updated comments.